Source/Orts.Simulation/Simulation/RollingStocks/SubSystems/CruiseControl.cs

```diff
@@ -0,0 +1,2277 @@
+                            Locomotive.SetDynamicBrakePercent(0);
+                        controllerVolts = 0;
+                        Locomotive.ThrottlePercent = 0;
+                        return;
+                    }
+                }
+
+
+                if ((SelectedMaxAccelerationStep == 0 && SelectedMaxAccelerationPercent == 0) || SpeedSelMode == SpeedSelectorMode.Start)
+                    WasForceReset = true;
+
+                if (SelectedMaxAccelerationPercent == 0 && SelectedMaxAccelerationStep == 0)
+                {
+                    WasBraking = false;
+                    if (SpeedRegMode == SpeedRegulatorMode.Auto && UseThrottleAsForceSelector) Locomotive.ThrottleController.SetPercent(0);
+                    Locomotive.SetThrottlePercent(0);
+                }
+                if (ForceResetRequiredAfterBraking && WasBraking && (SelectedMaxAccelerationStep > 0 || SelectedMaxAccelerationPercent > 0))
+                {
+                    Locomotive.SetThrottlePercent(0);
+                    controllerVolts = 0;
+                    maxForceN = 0;
+                    if (SpeedSelMode == SpeedSelectorMode.Parking)
+                        if (AbsWheelSpeedMpS < (SpeedIsMph ? MpS.FromMpH(ParkingBrakeEngageSpeed) : MpS.FromKpH(ParkingBrakeEngageSpeed)))
+                            Locomotive.SetEngineBrakePercent(ParkingBrakePercent);
+                    return;
+                }
+
+                if (ForceResetRequiredAfterBraking && !WasForceReset)
+                {
+                    Locomotive.SetThrottlePercent(0);
+                    controllerVolts = 0;
+                    maxForceN = 0;
+                    if (SpeedSelMode == SpeedSelectorMode.Parking)
+                        if (AbsWheelSpeedMpS < (SpeedIsMph ? MpS.FromMpH(ParkingBrakeEngageSpeed) : MpS.FromKpH(ParkingBrakeEngageSpeed)))
+                            Locomotive.SetEngineBrakePercent(ParkingBrakePercent);
+                    return;
+                }
+
+
+                if (canAddForce)
+                {
+                    if (Locomotive.AbsSpeedMpS == 0)
+                    {
+                        timeFromEngineMoved = 0;
+                        reducingForce = true;
+                    }
+                    else if (reducingForce)
+                    {
+
+                        timeFromEngineMoved += elapsedClockSeconds;
+                        float timeToReduce = Locomotive.SelectedTrainType == MSTSLocomotive.TrainType.Pax ? PowerReductionDelayPaxTrain : PowerReductionDelayCargoTrain;
+                        if (timeFromEngineMoved > timeToReduce)
+                            reducingForce = false;
+                    }
+                }
+                if (UseTrainBrakeAndDynBrake && CCIsUsingTrainBrake)
+                {
+                    canAddForce = true;
+                }
+                else if (Bar.FromPSI(Locomotive.BrakeSystem.BrakeLine1PressurePSI) < 4.98)
```

Comment:
Not acceptable for non UIC systems

Reply:
I see, but do we have in OR a variable defining which standard a consist adheres to? And what are the values for the american standard?

Reply:
You can use TrainBrakeMaxSystemPressure - 0.02bar instead

Reply:
Done, see next conversation